app/Models/Cts/Validation.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+namespace App\Models\Cts;
+
+use Illuminate\Database\Eloquent\Model;
+
+class Validation extends Model
+{
+    protected $connection = 'cts';
+    protected $guarded = [];
```

Comment:
Do you want to use fillable none instead? We shouldn't theoretically modifying anything.

Reply:
Good shout - resolved in commit b94d9250